### PR TITLE
Release 03-23-2021

### PR DESCRIPTION
**Pull request by benbcai**
### Summary
Release for:

- jest-config-terra 1.0.0
- terra-functional-testing 1.0.3
- terra-tooolkit-docs 1.8.3

### Additional Details

@cerner/terra

[CONTRIBUTORS.md]: ../blob/main/CONTRIBUTORS.md
[License]: ../blob/main/LICENSE

### CODE REVIEW

**Review by mjhenkes — APPROVED** [review]
Thanks for putting the release out!

File: lerna.json [hunk 1]
```diff
@@ -3,8 +3,7 @@
   "command": {
     "publish": {
       "ignoreChanges": [
-        "packages/package-json-lint/**",
-        "packages/react-config-terra/**"
```
Comment (StephenEsser) [inline lerna.json:7]: image

**Comment by benbcai** [comment]
> Thanks for putting the release out!

Thanks for the lerna.json update!

**Review by StephenEsser — APPROVED** [review]

File: lerna.json (hunk 1 above)
Comment (mjhenkes) [reply lerna.json:7]: Lerna still picks up changes in a 'private' repo then we get errors on publish where it won't let us push a private repo to public npm... Adding this for react-config-terra was overkill but i was being safe.

Comment (StephenEsser) [reply lerna.json:7]: What is `react-config-terra`?

Comment (mjhenkes) [reply lerna.json:7]: i think you'd call that a typo, lol

**Pull request merged**